tests/integration/fixtures/mutation-delete.json

```diff
@@ -2,45 +2,31 @@
   "data": {
     "a": {
       "clientMutationId": null,
-      "thing": {
-        "rowId": 1,
```

Comment:
Remember, if you add this field back also add these tests back. I think adding this field back actually would be super helpful because it’s the only way to see now deleted values like `note`.